CHANGELOG.md

```diff
@@ -1,5 +1,92 @@
 # Lotus changelog
 
+# 1.18.0-rc5 / 2022-11-1
```

Comment:
not meant to be here ?

Reply:
I think it is? Because the new version is v1.19.0-dev, so it's a bit odd if changelog is still 1.17.2? 

So i would say merge in 1.18.0-rc5 changelog now, and when we have the final release, update to 1.18.0?